terraform/ecs/main.tf

```diff
@@ -126,7 +126,7 @@ resource "aws_launch_template" "launchtemp" {
 }
 
 resource "aws_autoscaling_group" "clusterasg" {
-  name                = "clusterasg"
+  name_prefix         = "clusterasg"
```

Comment:
Can we use the testing id here or at least tag the auto scaling group with the testing ID?

Reply:
Added it and changed the other names